mozilla_django_oidc_db/models.py

```diff
@@ -249,10 +250,11 @@ class OpenIDConnectConfig(OpenIDConnectConfigBase):
         help_text=_("The name of the OIDC claim that is used as the username"),
     )
 
-    claim_mapping = models.JSONField(
+    claim_mapping = JSONField(
```

Comment:
Prevents you from providing invalid layout. Might cause issues with migrations with previously bad layout.

Maybe if it is wrong, we set it, to the default value.